runtime/binding-kafka/src/main/java/io/aklivity/zilla/runtime/binding/kafka/internal/cache/KafkaCacheCursorFactory.java

```diff
@@ -83,15 +83,15 @@ public KafkaCacheCursor newCursor(
 
     public final class KafkaCacheCursor implements AutoCloseable
     {
-        private final KafkaFilterCondition condition;
+        private KafkaFilterCondition condition;
         private final KafkaDeltaType deltaType;
         private final LongHashSet deltaKeyOffsets; // TODO: bounded LongHashCache, evict -> discard
 
         private Node segmentNode;
         private KafkaCacheSegment segment;
         public long filters;
         public long offset;
-        private long latestOffset;
+        public long latestOffset;
```

Comment:
With the proposed new `init` method described later, this change can be reverted.

Reply:
Also please add a blank line before public fields to enhance readability.